### PR TITLE
Net Neutrality Comments snippet.

### PR DESCRIPTION
**Pull request by ScottDowne**
@glogiotatidis Very very close to done.

It might get a few tweaks from a designer moving around a few pixels, and I didn't yet add in the opt out button which I plan on doing, but other than that I believe it's done.

Here is the sample YAML I used:

~~~
text: "It's time to act, the internet needs your voice. <a href='#'>Tell the FCC</a> to keep net neutrality intact."
~~~

Here is a screen shot of what it should look like: https://pageshot.net/bmkc2A683MbKn2KS/about

### CODE REVIEW

**Comment by ScottDowne** [comment]
@glogiotatidis I'm also leaving you a video of it in action via slack, because github wouldn't accept videos.

File: campaigns/net-neutrality-comments/snippet.html [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
...ZZUo6sFKitRDulRkNr0cEM0YGyrozhseYoMOt7kJJw9H4YoHXa1lxuDqlFs3dGCqC1p4g+n4kTJ5p5UCISh86RIvsEjisTQkk/cf0cieDK7ZoHaDli0AH2onVEpwCbA+to0EmnSfMHI9jEWjDygp2yISXFmi0gThkKpUwSMUh5jDXFCnIDX9So5Q1kOSbcjWIJnsG7kzYnC5biWXFQR67Nl1albwqYUzlbov2QAdEqBSbPxQpWkdGHnr+zZM+5wRVFJqz8G140oWbPVuIsnvkCoLXGrnOC4HcY3cV5vdQ1gKYb0kcnsB/SjtYpo2QZP0o9MErkj2642XXymV9hBATL5YboseacnCE/S2OiDK/1wn0tQ5c0RnpTa6/D5oAz6uHQNDBbbTDHwLWxLGVvic72crpvBJRL2Hxdxhyd5eSumYE0VzXcYq2pYBTJonODBFzbuMhEHSOhlDwsL5WFCxe+TGOXYM7GQen9BBjtGkAjtJvWHRFFaN3spq3qGyJPRhOJl1L/g6yBB2tFG2nQjYC0/k02Xi94wRoYtc5ogZPXwUsDVbxag1E2V/eZW2QhYieoy8rZh9bK61EHlCO6s+WPHR3cOi35JtscM2hdGqwOQEug9sBZnxP2BfbMUkmKQVveJFZqU3kd8Lh68MEHX1aIliu88POaMWPGevIG0jV3DaDbB5lC69AS9qR/y+WcLdUXuwGk3fDYgH+zpQ+ngLkUpJ944on1St57vsi5+6gNlqlq2aUZ6qLtxdLJmwz1zoF0xC0w1TYAzjr7W1S+KNWJ6wBzaptOwJlncVqD9uo66vQFOkGSDkhJaeVygbTA5+KlGY36QstrZP9nrDAApnAMwV/iulEqcGXXABrtpfWHJP19dUfekqRGTQFsNxh6aUQcHio0ZxkmG5U0IsZB/p6NJgzQKWki9xxSEkRK5sgZ9qKBKp3YGqLPJVLdxXIx9HR7GcUpAxYY5KMGwpLanXVsBKkcNnXD2+X5WgCdYtAy7zODtAboqE2A5ToEh+iCCFEXO0vqkAZQxApwAJjVp/AKAyV7kQh5Y0U3ARrLiEiVhk4NeZ7fa53Q61I20TSLxkOGKx2MCilwTjHnnLShDYYlrKxkGm/dxzo+zBsCkDuZlUX8+mVSKgYancchl1Cp5NrrMuaULSLKsC3WLLdngLZc4DQ4Y4FWi23hHqrBmvcRqfbC+4
```
Comment (glogiotatidis) [inline campaigns/net-neutrality-comments/snippet.html]: I'm curious why you need to use `setTimeout` here with no timeout?

**Comment by glogiotatidis** [comment]
I tried compressing the logo using tinypng to get the overall size of the template down and I did get a ~45% reduction. I cannot tell the difference but i guess the original image maybe visually better. Thoughts @ScottDowne ? 

Also I understand that the font is the major weight here but i thought to check this out anyway

**Comment by ScottDowne** [comment]
@glogiotatidis updated from some design tweaks, nothing major. Then added the opt out button.

I'll take a look at reducing the image size using tinypng.

**Comment by ScottDowne** [comment]
@glogiotatidis so I poked around for ways to reduce the file size. Starting with compressing the image.

I ended up reducing it by 100k by using a css filter to apply the grayscale to the ff logo, instead of new images :D

**Comment by glogiotatidis** [comment]
very clever, thanks for recusing size! 🐳

**Pull request merged**